dts/bindings/rtc/microcrystal,rv-8263-c8.yaml

```diff
@@ -0,0 +1,42 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2024 Daniel Kampert
+# Author: Daniel Kampert <DanielKampert@Kampis-Elektroecke.de>
+
+description: Micro Crystal RV-8263-C8 RTC
+
+compatible: "microcrystal,rv-8263-c8"
+
+include:
+  - name: rtc-device.yaml
+  - name: i2c-device.yaml
+
+properties:
+    int-gpios:
+      type: phandle-array
+
+    fast_mode:
```

Comment:
Okay, so I remove this binding.

Reply:
@bjarki-trackunit do you have an idea how to determine the mode bit? Because I don´t get any information about which step size should be used here. How is the interface normally used?